supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -11,5 +11,31 @@
 [[admonitions]]
 == Admonitions
 
+Admonitions should draw the reader’s attention to certain information. Keep admonitions to a minimum, and avoid placing multiple admonitions in close proximity to one another. If multiple admonitions are necessary, restructure the information by removing the less important statements and placing them into the flow of the main content.
+
+Allowed admonition types:
```

Comment:
I'm not crazy about `Allowed` as an adjective here. I suggest
`Permitted admonition types` or `Valid admonition types`

Reply:
+1

Reply:
+1